manticore/native/cpu/abstractcpu.py

```diff
@@ -589,6 +613,31 @@ def write_int(self, where, expression, size=None, force=False):
 
         self._publish('did_write_memory', where, expression, size)
 
+    def _raw_read(self, where, size=1):
+        '''
+        Selects bytes from memory. Attempts to do so faster than via read_bytes.
+
+        :param int where: address to read from
```

Comment:
TIL that it works this way too... but I'd move it to type annotation.
```suggestion
        :param where: address to read from
```